pytorch/training/docker/1.6.0/py3/Dockerfile.gpu

```
@@ -0,0 +1,184 @@
+ && ldconfig
+
+# Install Nvidia Apex
+RUN git clone https://github.com/NVIDIA/apex.git \
+ && cd apex \
+ && pip install -v --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./
+
+# Configure Open MPI and configure NCCL parameters
+RUN mv /home/.openmpi/bin/mpirun /home/.openmpi/bin/mpirun.real \
+ && echo '#!/bin/bash' > /home/.openmpi/bin/mpirun \
+ && echo 'mpirun.real --allow-run-as-root "$@"' >> /home/.openmpi/bin/mpirun \
+ && chmod a+x /home/.openmpi/bin/mpirun \
+ && echo "hwloc_base_binding_policy = none" >> /home/.openmpi/etc/openmpi-mca-params.conf \
+ && echo "rmaps_base_mapping_policy = slot" >> /home/.openmpi/etc/openmpi-mca-params.conf \
+ && echo "btl_tcp_if_exclude = lo,docker0" >> /home/.openmpi/etc/openmpi-mca-params.conf \
+ && echo NCCL_DEBUG=INFO >> /etc/nccl.conf \
+ && echo NCCL_SOCKET_IFNAME=^docker0 >> /etc/nccl.conf
+
+# Install OpenSSH for MPI to communicate between containers, allow OpenSSH to talk to containers without asking for confirmation
+RUN apt-get update \
+ && apt-get install -y  --allow-downgrades --allow-change-held-packages --no-install-recommends \
+ && apt-get install -y --no-install-recommends openssh-client openssh-server \
+ && mkdir -p /var/run/sshd \
+ && cat /etc/ssh/ssh_config | grep -v StrictHostKeyChecking > /etc/ssh/ssh_config.new \
+ && echo "    StrictHostKeyChecking no" >> /etc/ssh/ssh_config.new \
+ && mv /etc/ssh/ssh_config.new /etc/ssh/ssh_config \
+ && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /
+
+ADD https://raw.githubusercontent.com/aws/aws-deep-learning-containers-utils/master/deep_learning_container.py /usr/local/bin/deep_learning_container.py
+
+RUN chmod +x /usr/local/bin/start_with_right_hostname.sh \
+ && chmod +x /usr/local/bin/deep_learning_container.py \
+ && curl -o /license.txt https://aws-dlc-licenses.s3.amazonaws.com/pytorch-1.6.0/license.txt
```

Please add another revision to this PR changing the `curl` command here to a `wget`.

`curl -o <filename> <url>` doesn't return a non-zero error code when it fails to access the url.

`wget -O <filename> <uri>` returns a non-zero error code when it fails to access the url, telling us that the content at the url is either inaccessible or faulty.

---

If you know of a way to make `curl` throw an error instead of failing silently, that would be a good change as well, instead of changing `wget` to `curl`.

---

Addressed in next update.